pontoon/base/templatetags/helpers.py

```diff
@@ -34,6 +34,9 @@ def default(self, obj):
         if isinstance(obj, QuerySet):
             return list(map(str, obj))
 
+        if isinstance(obj, datetime.date):
+            return obj.isoformat()
+
```

Comment:
If we want to make this really really correct, then I believe this should go into a different class. This is not related to Django's LazyObjects at all. I'd write this I think: 

```python
class DatetimeAwareJSONEncoder(json.JSONEncoder):
    def default(self, obj):
        if isinstance(obj, datetime.date):
            return obj.isoformat()

class LazyObjectsJsonEncoder(DatetimeAwareJSONEncoder):
    # etc.
```